scripts/cli/tsconfig.json

```diff
@@ -5,6 +5,6 @@
   "extends": "../../tsconfig.json",
   "ts-node": {
     "transpileOnly": true,
-    "transpiler": "ts-node/transpilers/swc-experimental"
+    "swc": true
```

Comment:
This change has zero impact. I updated it to match the latest `ts-node` documentation.